homeassistant/components/zwave_js/config_flow.py

```diff
@@ -430,7 +431,7 @@ async def async_step_manual(
             step_id="manual", data_schema=get_manual_schema(user_input), errors=errors
         )
 
-    async def async_step_hassio(self, discovery_info: dict[str, Any]) -> FlowResult:
+    async def async_step_hassio(self, discovery_info: HassioServiceInfo) -> FlowResult:
```

Comment:
We haven't changed the access of `discovery_info` below. Are you planning to do that separately?

Reply:
I was planning to do it separately to keep the size of the PR as small as possible.

Reply:
The access has been updated.